tests/lib/restmail.js

```diff
@@ -7,15 +7,25 @@ define([
   'tests/lib/request',
   'intern/node_modules/dojo/Promise'
 ], function (request, Promise) {
+  var requestAttempts = 0;
+
   function waitForEmail(uri, number) {
     if (!number) {
       number = 1;
     }
-    console.log('Waiting for email at:', uri);
+
+    if (requestAttempts > 1) {
```

Comment:
My gut tells me that `2` might be a better value since in the current code there are pretty much always minimum 2 messages shown for each lookup.